### PR TITLE
[Remove Immediate] LPS-188559 Remove immediate property from DLProcessorRegistryImpl

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-188559

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:document

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:document

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

**Review by rafaprax — APPROVED** [review]

File: modules/apps/document-library/document-library-preview-audio-test/src/testIntegration/java/com/liferay/document/library/preview/audio/test/DLAudioFFMPEGAudioConverterTest.java [hunk 1]
```diff
@@ -72,35 +73,42 @@ public void testDoesNotGenerateAudioPreview() throws Exception {
...
+
+					Assert.assertFalse(
+						AudioProcessorUtil.hasAudio(
+							fileEntry.getFileVersion()));
+				});
+
+			List<LogEntry> logEntries = logCapture.getLogEntries();
+
+			Assert.assertTrue(
+				StringUtil.contains(
```
Comment (shuyangzhou) [inline modules/apps/document-library/document-library-preview-audio-test/src/testIntegration/java/com/liferay/document/library/preview/audio/test/DLAudioFFMPEGAudioConverterTest.java:99]: @MrJohn1911 This StringUtil.contains() usage is too weird, please change it to plain String contains assert.

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:document

**Pull request closed without merging**